### PR TITLE
Preparing rollup to use firebase-notifications

### PR DESCRIPTION
**Pull request by andreban**
- Added `rollup-plugin-node-resolve` to improve package resolution
- Added `rollup-plugin-commonjs` to make rollup compatible with
commonjs module.export style (used by firebase)

### CODE REVIEW

**Comment by andreban** [comment]
@ithinkihaveacat I'm adding this to pave the way to add firebase to the frontend. PTAL.

File: public/js/gulliver.es6.js [hunk 1]
```diff
@@ -6,9 +6,9 @@
 /* eslint-env browser */
 
 // As used by https://github.com/Financial-Times/polyfill-service/blob/master/polyfills/Promise/config.json
-import '../../node_modules/yaku/dist/yaku.browser.global.min.js';
+import 'yaku/dist/yaku.browser.global.min.js';
```
Comment (juliantoledo) [inline public/js/gulliver.es6.js]: What is this polyfill for?

Comment (andreban) [inline public/js/gulliver.es6.js]: @ithinkihaveacat Do you know this one?

Comment (andreban) [inline public/js/gulliver.es6.js]: It's a Promise polyfill. Added comments.

**Pull request merged**